apps/web/src/services/asset-inventory/collector/modules/CollectorItemInfo.vue

```diff
@@ -0,0 +1,236 @@
+                        height="1rem"
+                        width="1rem"
+                        color="inherit"
+                    />
+                </span>
+                <span class="icon-fill-wrapper">
+                    <p-i
+                        name="ic_check"
+                        class="check-icon"
+                        height="1rem"
+                        width="1rem"
+                        color="inherit"
+                    />
+                </span>
+                <span class="icon-fill-wrapper">
+                    <p-i
+                        name="ic_check"
+                        class="check-icon"
+                        height="1rem"
+                        width="1rem"
+                        color="inherit"
+                    />
+                </span>
+                <span class="icon-fill-wrapper">
+                    <p-i
+                        name="ic_check"
+                        class="check-icon"
+                        height="1rem"
+                        width="1rem"
+                        color="inherit"
+                    />
+                </span>
+                <span class="icon-fill-wrapper">
+                    <p-i
+                        name="ic_check"
+                        class="check-icon"
+                        height="1rem"
+                        width="1rem"
+                        color="inherit"
+                    />
+                </span>
+            </div>
+            <div class="to-history-detail">
+                <span>View history detail</span>
+                <p-i
+                    name="ic_chevron-right"
+                    width="0.75rem"
+                    height="0.75rem"
+                    color="inherit transparent"
+                />
+            </div>
+        </div>
+        <div
+            v-else-if="props.type === CollectorItemInfoType.SCHEDULE"
+            class="info-item"
+        >
+            <p class="info-label">
+                {{ props.label }}
+            </p>
+            <p-toggle-button
+                :value="state.toggleActive"
+                :label="state.toggleStatus"
+                :class="state.toggleActive ? 'toggle-active' : ''"
+                @change-toggle="handleChangeToggle"
+            />
+            <p-button style-type="transparent">
+                <p-i v-if="state.toggleActive"
+                     name="ic_edit"
+                     height="0.75rem"
+                     width="0.75rem"
+                     color="inherit"
+                     class="icon-schedule"
+                />
+                <p-i v-else
+                     name="ic_settings-filled"
+                     height="0.75rem"
+                     width="0.75rem"
+                     color="inherit"
+                     class="icon-schedule"
+                />
+                {{ state.toggleActive ? 'Edit' : 'Set' }} Schedule
+            </p-button>
```

Comment:
issue: It seems that the wrong component is used in this case.
<img width="777" alt="image" src="https://github.com/cloudforet-io/console/assets/26986739/3393a32f-02d5-4a56-8f42-b3743da16f5d">
We have p-text-button for this case, and before using the component, please refer to the [storybook](https://storybook.developer.spaceone.dev/?path=/docs/inputs-buttons-text-button--sizes).
That's because sometimes figma's component and the mirinae component is different each other. 🙂

But it seems that some components have problem in the storybook... I'll check it after reviewing this PR. 😭

Reply:
This is a temporarily written code without applying the data yet, so please review it again later!